mp3lyrics.rb

```diff
@@ -51,11 +83,22 @@ def to_b(string)
     puts "Fetching lyrics for #{artist} - #{title}"
     # Either no tag is set, the mp3 file has no USLT tag or we override anyway
     if !mp3.hastag2? || (mp3.hastag2? && !mp3.tag2.key?('USLT')) || override
-      lyrics = LyricWikia.new.get_lyrics(artist, title)
-      lyrics = Genius.new.get_lyrics(artist, title) if lyrics.nil?
-      lyrics = MetroLyrics.new.get_lyrics(artist, title) if lyrics.nil?
-      lyrics = AZLyrics.new.get_lyrics(artist, title) if lyrics.nil?
-      lyrics = SwiftLyrics.new.get_lyrics(artist, title) if lyrics.nil?
+      lyrics = nil
+      if wiki_to_use == nil or wiki_to_use == "lyricwikia"
```

Comment:
use `wiki_to_use.nil?` and single quotes. Why `or` over `||`
(applies to the rest, too)